tests/parsing/GlobbingTest.py

```diff
@@ -218,7 +219,8 @@ def setUp(self):
     def _test_glob(self, pattern, file_list):
         results = sorted([os.path.normcase(g) for g in glob(pattern)])
         file_list = sorted([os.path.normcase(f) for f in file_list])
-        self.assertEqual([i for i in results if "__pycache__" not in i],
+        self.assertEqual([i for i in results if re.search(r"(__pycache__|.pyc)",
```

Comment:
BWT did we not a unspoken policy of not having single character variables? I remember reading this somewhere. Not sure though.